projects/dev/src/app/group/group.module.ts

```diff
@@ -22,6 +24,13 @@ import { PendingRequestComponent } from './group-manage/pending-request/pending-
     GroupManageComponent,
     PendingRequestComponent,
   ],
-  imports: [CommonModule, GroupRoutingModule, TableModule, CoreModule],
+  imports: [
+    CommonModule,
+    GroupRoutingModule,
+    TableModule,
+    ProgressSpinnerModule,
+    ToastModule,
```

Comment:
Why is it declared in `groups.module`?

Reply:
Fixed, moved those modules to `core` modules for further use.

Reply:
Please run `ng build core` before you test.

Reply:
I still see  `ToastModule` in `projects/dev/src/app/group/group.module.ts`

Reply:
Could you check again?
It's already removed.
🤔

Reply:
Now that's ok. Maybe you didn't push.